src/App.js

```diff
@@ -1,35 +1,38 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import { BrowserRouter, Route } from "react-router-dom";
+import "./App.css";
+import Header from "./components/Header";
+import Home from "./components/Home";
+import Project from "./components/Project";
+import WorksList from "./components/WorksList";
+import data from "./data.json";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
+    let project;
+    data.forEach((proj) => {
+        return (project = proj);
+    });
+    let { id, year, title, img, technologies } = project;
+    return (
+        <BrowserRouter>
+            <div className="App">
+                <header className="App-header">
+                    <Header />
+                </header>
+                <main className="App-main">
+                    <Route exact path="/" component={Home} />
+                    <Route exact path="/" component={WorksList} />
```

Comment:
this will show both the home and works list components for the same route `/` .. if that is what you want , you should just move your Worklist component to your Home component and just render the home